esp8266-fastled-webserver/Noise.cpp

```diff
@@ -145,14 +145,114 @@ void drawNoise(CRGBPalette16 palette, uint8_t hueReduce = 0)
   noisez += noisespeedz;
 }
 
-#if HAS_POLAR_COORDS // change to "HAS_CONCENTRIC_RINGS" ?
+void rainbowNoise() {
```

Comment:
You wrote:

> Thoughts on consolidating HAS_POLAR_COORDS and HAS_RADIUS_PROXY completely?

Yes, I think removal of the defines for `HAS_POLAR_COORDS` and `HAS_RADIUS_PROXY` would be a good thing.  Any board that defines `HAS_COORDINATE_MAP` should simply define the following:
* `coordsX[NUM_PIXELS]`
* `coordsY[NUM_PIXELS]`
* `angles[NUM_PIXELS]`
* `radiusProxy[NUM_PIXELS]`
* `RADII_SCALE_DIVISOR` and `RADII_SCALE_MULTIPLIER` -- but see next comment on removal of these?

Of course, `radiusProxy` could normally still be a reference to an existing array, to avoid double-storage costs.

At the same time, I think keeping the `radiusProxy` variable named as such would be good for at least two reasons:

1. It makes it clear that the radius proxy does not have to be the X/Y values converted to a radius.  For some props, it's **_better_** to have a manually defined proxy for the radius value, rather than having it hard-coded to use the actual radius values.
2. It also avoids any confusion in later-written effects, which might otherwise expect `radiusProxy[i]` to mean the actual offset from center for the pixel.

I think Kraken64 may not use the **_actual_** radius for the pixels.  In essence, it's an example of the above.